core/templates/pages/exploration-editor-page/improvements-tab/improvements-tab.directive.html

```diff
@@ -1,66 +1,3 @@
-            style="margin: 0">
-          <[getOpenTaskCount()]>
-        </h1>
-      </div>
-      <div class="options-card" style="padding: 10px; margin: auto">
-        <input ng-model="onlyShowOpenTasks" ng-init="true" type="checkbox"
-               class="protractor-test-improvements-only-open-input">
-        <span style="font-size: 14px">
-          Show only "Open" tasks.
-        </span>
-      </div>
-    </md-card>
-  </div>
-
-  <md-card ng-repeat="task in getTasks() | orderBy:getTaskTitle:true | filter:isTaskShown"
-           ng-if="!isTaskObsolete(task)"
-           ng-class="[task.getDirectiveType(), {'oppia-improvement-task': isTaskOpen(task), 'oppia-improvement-closed-task': !isTaskOpen(task)}]"
-           class="protractor-test-improvements-task">
-
-    <div class="oppia-improvement-task-header modal-header">
-      <h2 class="oppia-improvement-task-header-title protractor-test-improvements-task-title">
-        <[task.getTitle()]>
-      </h2>
-
-      <div class="oppia-improvement-task-header-status-pill">
-        <span ng-class="getStatusCssClass(task.getStatus())"
-              class="protractor-test-improvements-task-status">
-          <[getHumanReadableStatus(task.getStatus())]>
-        </span>
-      </div>
-    </div>
-
-    <div ng-switch="task.getDirectiveType()" class="modal-body">
-      <answer-details-improvement-task ng-switch-when="answer-details"
-                                       data="task.getDirectiveData()"
-                                       class="protractor-test-improvements-task-body">
-      </answer-details-improvement-task>
-      <feedback-improvement-task ng-switch-when="feedback" data="task.getDirectiveData()"
-                                 class="protractor-test-improvements-task-body">
-      </feedback-improvement-task>
-      <playthrough-improvement-task ng-switch-when="playthrough" data="task.getDirectiveData()"
-                                    class="protractor-test-improvements-task-body">
-      </playthrough-improvement-task>
-      <suggestion-improvement-task ng-switch-when="suggestion" data="task.getDirectiveData()"
-                                   class="protractor-test-improvements-task-body">
-      </suggestion-improvement-task>
-    </div>
-
-    <div ng-if="task.getActionButtons().length > 0"
-         class="oppia-improvement-task-footer modal-footer">
-      <button ng-repeat="actionButton in task.getActionButtons()"
-              ng-click="actionButton.execute()" ng-disabled="!actionButton.isEnabled()"
-              ng-class="actionButton.getCssClass()"
-              class="btn protractor-test-improvements-action-button"
-              style="margin-left: 15px" type="button">
-        <[actionButton.getText()]>
-      </button>
-    </div>
-  </md-card>
+<div class="oppia-dashboard-container">
+  <!-- TODO(brianrodri): Introduce new changes for improvements tab. -->
```

Comment:
Do we still need this?

Reply:
Nope, but the exact filename would be reintroduced so I don't think keeping it would do any harm (https://github.com/brianrodri/oppia/pull/18)

Reply:
So the logic for selecting the improvements tab is still helpful and will stay, so I've kept these directives to make sure that logic remains relevant.

Reply:
sg, thanks